docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,593 @@
+   that was created in step one above when you cloned the repo). You can read more about this in the
+   `Install Brain-Score Repos and Dependencies`_ section below. The command for a terminal is: ::
+     pip install .
+3. Specify the model to test. In this example, we can
+   just use the model defined in ``pytorch.py``, located in the ``examples`` folder
+   of this repository. More info on this step can be found in
+   the `Submitting a Model to Brain-Score.org Part 1: Preparing the Model`_ section of
+   this guide. In essence, you need to implement the functions outlined in
+   the ``sample-model-submission/models/base_models.py`` file.
+4. Test the model on your machine. You can do this simply by running the ``pytorch.py`` file
+   (again, located in the ``examples`` folder)
+   or the file where you implemented
+   the ``base_models.py`` template. If you followed the steps correctly, you should
+   receive a message on the Python console indicating that you are ready to submit.
+5. Submit to ``brain-score.org``. This step is slightly brittle as of now,
+   and is easy to do wrong; we recommend skipping down to
+   `Submitting a Model to Brain-Score.org Part 2: Upload`_
+   section to see the structure of the zip file that
+   our site needs in order to process your submission.
+   That’s it! Read more below to get a better idea of the process, or to help fix bugs that might come up.
+
+
+Overview
+========
+
+Brain-Score sounds great! How do I get started? What do I do next? How do I submit?
+--------------------------------------------------------------------
+
+Start here if you have already
+(or would like to create) a model to submit and get a Brain-Score.
+This guide will walk you through from downloading our code to receiving a
+submission score from the website. Completion of the tutorial
+should take about 15-20 minutes.
+
+What does Brain-Score look like, exactly?
+--------------------------------------------------------------------
+The main code for Brain-Score is contained and hosted on `Github`_
+on various repos, free for anyone to fork,
+clone, or download. The main page has 10 repos (9 visible to non-devs)
+that make up the entire source code base- this can be overwhelming, but
+fear not. We will instroduce you to each one, as well as explain that
+you most likely will only need a single repo to get a model up and scored. It is
+good to see, however, the structure of how Brain-Score operates with its
+various components. They are:
+
+1. ``brain-score``: the heart of the code that runs analysis and comparisons.
+2. ``sample-model-submission``: template and examples for model submissions.
+3. ``candidate_models``: various pre-trained models/models that have already been scored.
+4. ``model-tools``: helper functions that translate from machine learning models
+   to brain models to be tested on brain-score.
+5. ``brainio_collection``: the repo that packages and collects the stimuli/data.
```

Comment:
Has references to `brainio_collection` and `brainio_base`.  But I'm planning to take care of that in another PR.

Reply:
I resolved the conflicts, so we should be good :)